backend/static_crawler.py

```diff
@@ -0,0 +1,51 @@
+from json import load, dump, dumps
+from pathlib import Path
+
+from kaggle import download, validate, SkinCancerMnist
+from isic import ISIC_getdata
+
+with open('config.json', 'r') as f:
+    data = load(f)
+    host = data['host']
+
+
+def get_kaggle_full(n_images):
+    """Download data from Kaggle."""
+    full_data = []
+
+    if validate():
+        dataset = 'dinhanhx/skin-cancer-mnist-ham10000'
+        path = Path('./static/kaggle')
+        dataset_path = download(dataset, path)
+        scm = SkinCancerMnist(dataset_path)
+        for i in range(scm.get_len_dataset()):
+            full_data.append(scm.get_img_metadata(i))
+
+    return full_data
```

Comment:
Parameter `n_images` is never used. I suggest you to add an if else statement to check whether it's bigger than the length of SkinCancerMnist dataset. If it's bigger, set `n_images` equal to the length.

Reply:
Oh I copied from the isic one. I'll delete it.